wire-ios-sync-engine/Source/Synchronization/Transcoders/ZMMissingUpdateEventsTranscoder.m

```diff
@@ -165,6 +165,7 @@ - (NSUUID *)processUpdateEventsAndReturnLastNotificationIDFromResponse:(ZMTransp
     NSMutableArray<ZMUpdateEvent *> *parsedEvents = [NSMutableArray array];
     NSMutableArray<NSUUID *> *eventIds = [NSMutableArray array];
     NSUUID *latestEventId = nil;
+    // TODO: double check this
     ZMUpdateEventSource source = self.isFetchingStreamForAPNS || self.isFetchingStreamInBackground ? ZMUpdateEventSourcePushNotification : ZMUpdateEventSourceDownload;
```

Comment:
is this correct that isFetchingStreamInBackground would be mark as ZMUpdateEventSourcePushNotification?